lightning/src/ln/peers/handshake/acts.rs

```diff
@@ -0,0 +1,42 @@
+pub const ACT_ONE_LENGTH: usize = 50;
+pub const ACT_TWO_LENGTH: usize = 50;
+pub const ACT_THREE_LENGTH: usize = 66;
```

Comment:
There are still places in mod.rs where these aren't used.

Reply:
In places where it's not a concrete act, but either 1 or 2, they shouldn't be used. But I found one spot to replace